bluetooth/src/androidLibMain/kotlin/device/DeviceConnectionManager.kt

```diff
@@ -78,6 +79,13 @@ internal actual class DeviceConnectionManager(
             }
         }
 
+        override fun onMtuChanged(gatt: BluetoothGatt?, mtu: Int, status: Int) {
```

Comment:
it's kind of weird to have gatt param only for debug purpose 🤔

Reply:
It's from Android...